packages/node-sdk/example/bucket.ts

```diff
@@ -0,0 +1,8 @@
+import { BucketClient } from "../src";
+
+export const bucket = new BucketClient({
+  secretKey: process.env.BUCKET_SECRET_KEY || "",
```

Comment:
We need to settle on the fallback flags and maybe add some here for reference?

Reply:
yes, good idea